.github/workflows/container-release.yml

```diff
@@ -52,28 +51,52 @@ jobs:
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
 
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: quay.io/invidious/invidious
+          tags: |
+            type=sha,format=short
+            type=sha,format=short,prefix={{date 'YYYY.MM.DD'}}-,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
+          labels: |
+            quay.expires-after=12w
+
       - name: Build and push Docker AMD64 image for Push Event
-        if: github.ref == 'refs/heads/master'
```

Comment:
Why would you remove that? We don't want to build and push containers for other branches!